build-system/tasks/storybook/index.js

```diff
@@ -108,7 +107,8 @@ async function storybook() {
   if (!build) {
     createCtrlcHandler('storybook');
   }
-  envs.map(build ? buildEnv : launchEnv);
+  const buildDir = build === true ? 'examples/storybook' : build;
+  envs.map(buildDir ? (env) => buildEnv(env, buildDir) : launchEnv);
```

Comment:
It took me a while to realize that the `--build` flag is being overloaded to be either a boolean or a string. For maintainability / usability, I think it's better to add a separate `--build_dir` flag that works in conjunction with `--build`.